### ISSUE TITLE
I can't open the file

### ISSUE THREAD

**Issue by WhiteTiger1999** [SamusAranX/SH2Unpack#2]
I get the error "Can't open file: The system cannot find the path specified". Although the path is specified correctly

**Comment by SamusAranX** [comment]
I need more details for this to be at all actionable.
* What OS are you running?
* What exact command did you run?

**Comment by WhiteTiger1999** [comment]
I am using Win10 x64. I entered the command unpack. For a more detailed idea of ​​the error, I can send a screenshot

**Comment by SamusAranX** [comment]
Please paste the exact command you used. 
Run it again, select and copy the command from the command line window, then paste it here.

**Comment by WhiteTiger1999** [comment]
C:\SH2Unpack\sh2unpack-windows-x64 unpack -i C:/sh2/SILENT_HILL_2_V0_10 C:/SH2Extract/

Here is a screenshot of the entire operation
![Desktop 02-10-2024 11-23-31-532](https://github.com/user-attachments/assets/b0b3e473-452c-4117-9f6c-7fa62431f15f)

**Comment by SamusAranX** [comment]
That error message is coming from the file open call itself, and if it says the file's not there, the file's not there. It looks like you've been manually typing in paths, so try not doing that and instead copy/pasting file paths directly.

**Comment by WhiteTiger1999** [comment]
Yes, I tried pasting the copied path. And that was also unsuccessful. Maybe I missed something, what do you think?
